lib/service/Service.ts

```diff
@@ -209,10 +213,16 @@ class Service extends EventEmitter {
   }
 
   /*
-   * Subscribe to incoming peer orders.
+   * Subscribe to peer order events.
    */
   public subscribePeerOrders = async (callback: Function) => {
-    this.orderBook.on('peerOrder', order => callback(order));
+    this.orderBook.on('peerOrder.incoming', order => callback(order));
+    this.orderBook.on('peerOrder.invalidation', order => callback({
+      canceled: true,
```

Comment:
I think this interface design is a bit problematic.
What about having an enum, or boolean, to represent the event type, and then the payload? 
`{ type/isCancellation, payload }`

We can discuss that separately, for a different PR. Don't let this delay you from merging, if it's blocking #266

Reply:
I was thinking that we will rework the gRPC messages in the alpha stage anyway and just adding the `canceled` boolean was the simplest solution so that it can get merged for prealpha.

But I agree on the new design you described. Could you write it down in an issue so that we don't forget about it?

Reply:
Yep, sure.

Reply:
Just did that: https://github.com/ExchangeUnion/xud/issues/298